internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -191,8 +191,10 @@ func (e *EnvStackConfig) telemetryConfig() *template.Telemetry {
 
 func (e *EnvStackConfig) importCertARNs() []string {
 	// If a manifest is present, it is the only place we look at.
+	// Look under both public and private.
 	if e.in.Mft != nil {
-		return e.in.Mft.HTTPConfig.Public.Certificates
+		allCerts := append(e.in.Mft.HTTPConfig.Public.Certificates, e.in.Mft.HTTPConfig.Private.Certificates...)
+		return allCerts
```

Comment:
Okay, but the MVP behavior is still to consider the certs public unless there are no public subnets, in which case we consider them private...right?

Reply:
Yup yup - that's the end behavior that we'll maintain for the MVP. 

The comment was just to re-structure the `stack/env.go` and `template` files to split `ImportCertArns` into two so that once we have the env manifest, we have a path for enabling certs on both load balancers.

I'm 🆗 w keeping it like this in this PR so that it doesn't become v large, if we can send the follow-up PR with imported certs split?